src/index.html

```diff
@@ -12,5 +12,33 @@
   </head>
   <body>
     <h1>Search bar airbnb</h1>
+    
+    <form 
+      class="form"
+      action="#" 
+      method="post"
+      data-qa="big"
+    >
+      <input 
+        class="input"
+        type="search" 
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+    </form>
+   
+
+    <form 
+      class="form form-small"
```

Comment:
`small` is modifier, so should be `form--small` or `form_small`. also fix for `input`